### PR TITLE
Update Blueprints index.rst

### PR DESCRIPTION
**Pull request by Niehaus**
Update the layout of Blueprints #1180, it should look like this example now:

Screenshot_2020-10-29 Blueprints — Kytos SDN Platform 2020 2b1 documentation

### CODE REVIEW

File: docs/blueprints/make_blueprints_table.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    
+    
+    table_cell = ''
+    for fp in glob.glob(f'{directory}/EP*.rst'):
+        split_dir = ''.join(fp.split('./blueprints/'))
+        bp_rst = ''.join(split_dir.split('.rst'))
+    
+        with open(fp) as origin_file:
+            title = 'Metadata not define for :Title:'
+            status = 'Metadata not define for :Status:'
```
Comment (hdiogenes) [inline docs/blueprints/make_blueprints_table.py]:
```suggestion
            title = ''
            status = ''
```

File: docs/blueprints/make_blueprints_table.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    
+    return file_header, toc_tree_text
+    
+    
+def create_table(directory):
+    #Create the table header
+    table_header = ''
+    table_header += '+' + '-' * 23 + '+' + '-' * 82 + '+' + '-' * 20 + '+\n'
+    table_header += '|' + ' Blueprint'  + ' ' * 13 + '|' + ' Title' + ' ' * 76 + '|' + ' Status' + ' ' * 13 + '|\n'
+    table_header += '+' + '=' * 23 + '+' + '=' * 82 + '+' + '=' * 20 + '+\n'
```
Comment (hdiogenes) [inline docs/blueprints/make_blueprints_table.py]: Please use a global string for this, disable the lint 80-char check if necessary.

File: docs/blueprints/make_blueprints_table.py [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+                    title = ''.join(title.split("\n"))
+                if re.findall(r':Status:', line):
+                    status = ''.join(line.split(':Status:'))
+                    status = ''.join(status.split("\n"))
+                    break
+                
+        title_space = 82 - (len(title))
+        status_space = 20 - (len(status))
+        table_cell += '|' + ' :doc:`'+ bp_rst + '<' + bp_rst + '/>` '  + ' ' + '|' + title + ' ' * title_space + '|' + status + ' ' * status_space + '|\n'
+        table_cell += '+' + '-' * 23 + '+' + '-' * 82 + '+' + '-' * 20 + '+\n'
```
Comment (hdiogenes) [inline docs/blueprints/make_blueprints_table.py]: I think it's better to just use a string template here.

File: docs/blueprints/index.rst [hunk 4]
```diff
@@ -4,8 +4,58 @@
...
 
+Table of Blueprints
+
 .. toctree::
-  :caption: List of Blueprints
+  :hidden:
   :glob:
 
   EP*
+
```
Comment (hdiogenes) [inline docs/blueprints/index.rst]: This can be generated in the build stage of the docs.

File: docs/blueprints/make_blueprints_table.py (hunk 3 above)
Comment (Niehaus) [reply docs/blueprints/make_blueprints_table.py]: I'm still working on some changes, but I'll take a look in your suggestions in order to apply them

**Pull request merged**